diremanwolf/CLI-multiple-themes
# Feat: add-CLI

#### What does this PR do?
This PR includes node packages, changes to project's core and two components, *Reader* and *Compiler*.

#### Where should the reviewer start?
* `src/index.js`
* `src/components/Reader/Reader.js`
* `src/components/Compiler/Compiler.js`

#### How should this be manually tested?
* Clone this branch
* Run `$ npm install`
* Run `$ node index --help`; it should deploy on the terminal, the information of **CLI-multiple-themes** package and its usage.

#### Any background context you want to provide?
It's a WIP; I'm currently mind-blocked.
